CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ From v1.0.0 onwards, this project adheres to [Semantic Versioning](https://semve
 
 - Up to date with releases.
```

Comment:
Technically not up to date with releases after this is merged.

Reply:
Please advise.

Reply:
You can replace "Up to date with releases" with your changelog message and we'll do another PR to correct the changelog (move your change to the correct version) after release

Reply:
I can fix this